dev/build.image/profiles/microProfile7/features.xml

```diff
@@ -1,4 +1,5 @@
 <feature>microProfile-7.0</feature>
+<feature>microProfile-7.1</feature>
```

Comment:
For both the `microProfile7` and `jakartaee11` profiles you will now need to add or update a `packageServerConflict` property to their package tasks in `build.image/build.gradle` so that they package correctly now that they have conflicting MP features.  You can see an example in #26581, but will need to do the right conflicts for MP 7 and 7.1.